Automattic/wp-calypso
# Stats: Update main chart area styling

#### Proposed Changes

<img width="1081" alt="截圖 2022-10-29 上午3 58 34" src="https://user-images.githubusercontent.com/6869813/198722342-03444eda-cae3-4dbd-8a5f-1e887d953486.png">

* Introduce feature flag `stats/new-main-chart` in components for new styles to avoid affecting existing page styles.
* Introduce feature flag `stats/new-main-chart` for the new main chart structure.
* Update main chart styles in line with the Design.

#### Testing Instructions



> Note: There are a few edge case styles that need feedback from the Design team [marked in the issue](https://github.com/Automattic/wp-calypso/issues/69041#issuecomment-1293854667).

* Launch the `local` development application.
* Navigate to page `/stats/week/${your-site}`.
* Ensure styling on the `Traffic` tab page is aligned with the `After` version of Design.
* Ensure the `primary color` styling is applied according to themes.
* Modify `stats/new-main-chart` in development.json to `false`.
* Ensure styling on the `Traffic` tab page is recovered to the `Before` version.




#### Pre-merge Checklist






Related to #69041
